core/src/avm2/globals/xml.rs

```diff
@@ -164,13 +164,26 @@ pub fn parent<'gc>(
 pub fn elements<'gc>(
     activation: &mut Activation<'_, 'gc>,
     this: Option<Object<'gc>>,
-    _args: &[Value<'gc>],
+    args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
     let xml = this.unwrap().as_xml_object().unwrap();
+    let element_name = if args[0] == Value::Undefined {
+        AvmString::from("*")
```

Comment:
Are you sure this is the correct behavior if `XML.elements(undefined)` is called? Otherwise I don't think `XML.elements()` can be called without arguments.